mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -504,21 +506,60 @@ public void testCreateTableWithNotSupportedTypesWithAutoConversion() {
     TableIdentifier identifier = TableIdentifier.of("default", "not_supported_types");
     // Can not create INTERVAL types from normal create table, so leave them out from this test
     Map<String, Type> notSupportedTypes = ImmutableMap.of(
-        "TINYINT", Types.IntegerType.get(),
-        "SMALLINT", Types.IntegerType.get(),
-        "VARCHAR(1)", Types.StringType.get(),
-        "CHAR(1)", Types.StringType.get());
+            "TINYINT", Types.IntegerType.get(),
+            "SMALLINT", Types.IntegerType.get(),
+            "VARCHAR(1)", Types.StringType.get(),
+            "CHAR(1)", Types.StringType.get());
```

Comment:
Can you revert these whitespace-only changes?

Reply:
Done.